backend/tournaments/migrations/0001_initial.py

```diff
@@ -15,35 +15,35 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
```

Comment:
I regenerated this because there was an error in model. (it's automatically generated so no need to review it)
sqlite database file needs to be deleted and recreated.
(more "correct" way would be to generate second mirgration which fixes database but I think it's better to just recreate it as we don't have prod database yet)